src/blst-native/secretKey.ts

```diff
@@ -1,5 +1,5 @@
 import blst from "@chainsafe/blst";
-import crypto from "crypto";
+import {randomBytes} from "@noble/hashes/utils";
```

Comment:
I'm fine with the change but if this is only changing things on the node side that do not get used in browser, why would there be a polyfill used?  Wouldn't the code be running the native node code because its running in node?